### DISCUSSION TITLE
Ignite and mono-repo (Turborepo react-native starter)

### DISCUSSION THREAD

**Discussion by musabgulfam** [infinitered/ignite#2872] [Q&A] [answered]
Hi there,
So I have been tasked with building an already existing ignite web/mobile app project to a mono-repo, since we want the native Firebase wrappings for iOS and Android and use Firebase JS SDK for web.

I am a little bit confused how to proceed with this, do we issue the command `npx ignite-cli@latest new my-app` in `/apps` directory after running the command `npx create-turbo@latest -e with-react-native-web`? More importantly, is there any documentation that outlines this specific use-case? Since I was unable to find one.

Thanks in advance.

**Comment by frankcalise** [comment, marked as answer]
Hi I converted this to a discussion since it falls under Q&A more than an issue. 

We have some monorepo documentation on the Ignite Cookbook available here:
- https://ignitecookbook.com/docs/recipes/MonoreposOverview/
- https://ignitecookbook.com/docs/recipes/SettingUpYarnMonorepo/

Although those don't specifically cover Turbo, an example of a Turbo repo with an Ignite app inside of it can be seen here: https://github.com/infinitered/react-native-mlkit

As far as where to issue your new command, I don't think it matters. You could do it outside of the directory and copy the files in or you could do it inside. One thing I would mention is you probably don't want to be installing dependencies from Ignite's CLI. So I'd make sure you would answer `No` to that prompt or pass `--install-deps=false`.

Then run your install you normally would within a Turbo repo

**Reply by musabgulfam** [reply]
Thank you very much.
